vendor/github.com/hashicorp/vault-plugin-secrets-ad/plugin/path_config.go

```diff
@@ -54,7 +54,7 @@ func (b *backend) pathConfig() *framework.Path {
 }
 
 func (b *backend) configFields() map[string]*framework.FieldSchema {
-	fields := ldaputil.ConfigFields()
+	fields := ldaputil.ConfigFields(false)
```

Comment:
Don't forget to move this into the actual plugin

Reply:
Yep, that will come!